src/routes/task/task.ts

```diff
@@ -1,24 +1,27 @@
 const router = require('express').Router()

```

Comment:
aby łatwiej było rozpoznać, że w danym pliku są endpointy dodaj do niego przypisane daj mu rozszerzenie `routes` czyli w tym wypadku to powinno byc `task.routes.ts`. Zaimplementuj tak dla pozostałych plików z endpointami